api/SVGGeometryElement.json

```diff
@@ -4,42 +4,132 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGeometryElement",
         "support": {
-          "chrome": {
-            "version_added": "1"
-          },
-          "chrome_android": {
-            "version_added": "18"
-          },
-          "edge": {
-            "version_added": "≤79"
-          },
-          "firefox": {
-            "version_added": "53"
-          },
-          "firefox_android": {
-            "version_added": "53"
-          },
+          "chrome": [
+            {
+              "version_added": "56"
+            },
+            {
+              "version_added": "1",
+              "version_removed": "56",
+              "partial_implementation": true,
+              "notes": "The <code>SVGGeometryElement</code> interface itself is not present, but some of its members are available on the <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a> interface."
+            }
+          ],
+          "chrome_android": [
+            {
+              "version_added": "56"
+            },
+            {
+              "version_added": "18",
+              "version_removed": "56",
+              "partial_implementation": true,
+              "notes": "The <code>SVGGeometryElement</code> interface itself is not present, but some of its members are available on the <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a> interface."
+            }
+          ],
+          "edge": [
+            {
+              "version_added": "79"
+            },
+            {
+              "version_added": "12",
+              "version_removed": "79",
+              "partial_implementation": true,
+              "notes": "The <code>SVGGeometryElement</code> interface itself is not present, but some of its members are available on the <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a> interface."
+            }
+          ],
+          "firefox": [
+            {
+              "version_added": "53"
```

Comment:
This is when `SVGGeometryElement` itself was introduced in Firefox. It was only later that all interfaces were made to inherit from it. But doing that as 3 entries here does not seem like it would make the situation more clear to anyone. The subfeatures are what matter more here.

Reply:
Great. I like this pragmatic approach. 👍